v3/tasks/20241205-v3-batch-router/readme.md

```diff
@@ -0,0 +1,10 @@
+# V3 Batch Router
```

Comment:
```suggestion
# 2024-12-05 - V3 Batch Router
```
Didn't notice this on the pools (should do a follow-up for consistency), but we have always included the date in the readme. Here and elsewhere.